tasks/kibana.yml

```diff
@@ -1,15 +1,23 @@
 ---
-- name: Download Kibana tarball.
-  get_url:
-    url: https://download.elastic.co/kibana/kibana/kibana-{{ elk_kibana_version }}-linux-x64.tar.gz
-    dest: /opt/kibana-{{ elk_kibana_version }}-linux-x64.tar.gz
-    # Don't download file every time, only if it doesn't exist.
-    force: no
+- name: Add Kibana repo apt key.
+  apt_key:
+    url: https://packages.elastic.co/GPG-KEY-elasticsearch
```

Comment:
Good catch. Let's definitely keep the config DRY, so consolidate the apt_key import into a single task. It'd probably be best housed in `tasks/packages.yml`, since it's used by all three components of the ELK stack.